integration-tests/go.mod

```diff
@@ -47,6 +49,8 @@ require (
 	golang.org/x/sync v0.6.0
 	golang.org/x/text v0.14.0
 	gopkg.in/guregu/null.v4 v4.0.0
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
@skudasov yes, I think so. Let me update it

Reply:
@skudasov I think we have to have this `gopkg.in/yaml.v2`. This module is used to parse yaml file like .github/e2e-tests.yml. I want to use yml format for it since it is part of Github CI convention.